docs/docs/quickstart/real-world-example/README.md

```diff
@@ -0,0 +1,246 @@
+
+![image-20210519012640800](./images/image-20210519012640800.png)
+
+ The action we will use accepts the array of blocks, so let's extract that and export a populated array from our code step (i.e., we don't need to generate the entire JSON payload).
+
+Add a step to **Run Node.js code** and name it `steps.generate_slack_blocks`. 
+
+![image-20210518201050946](./images/image-20210518201050946.png)
+
+Next, add the following code to `steps.generate_slack_blocks`:
+
+```javascript
+// Require iso-639-1 to convert language codes into human readable names
+const ISO6391 = require('iso-639-1')
+
+// Require lodash to help extract values from intermittent fields in the Tweet object
+const _ = require('lodash') 
+
+// Function to return a friendly language name (or "Unknown") for ISO language codes
+function getLanguageName(isocode) {
+  try { return ISO6391.getName(isocode) } 
+	catch (err) { return 'Unknown' }
+}
+
+// Function to format numbers over 1000 -- from Stack Overflow https://stackoverflow.com/questions/9461621/format-a-number-as-2-5k-if-a-thousand-or-more-otherwise-900
```

Comment:
Technically you can use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat, as well: https://pipedream.com/@dylburger/format-numbers-p_jmCKlvz/edit